From 69144dff82f5aa58e10816fa73e77177d7b6e41a Mon Sep 17 00:00:00 2001
From: Eric Sun <jian.sun@freescale.com>
Date: Tue, 11 Sep 2012 09:53:33 +0800
Subject: [PATCH 349/449] ENGR00218972-1 Add ddr location description in
 readme

The default ddr location parameter for MX6Q/DL/SL ARM2 platform is
provided in the README file

Signed-off-by: Eric Sun <jian.sun@freescale.com>
---
 test/mxc_secureboot/README | 22 ++++++++++++++++++++--
 1 file changed, 20 insertions(+), 2 deletions(-)

diff --git a/test/mxc_secureboot/README b/test/mxc_secureboot/README
index e89719c..3ce6ab3 100644
--- a/test/mxc_secureboot/README
+++ b/test/mxc_secureboot/README
@@ -24,7 +24,7 @@ File Description
 	    . "./mk_secure_uboot 0x87800000"
 	        - where 0x87800000 is the DDR location where u-boot is
 		  copied to by ROM code, different from platform to platform
-	    . u-boot-signed.bin will be generated. It is the u-boot image
+	    . u-boot-signed-pad.bin will be generated. It is the u-boot image
 	      with signature
 
 	For generating secure uImage
@@ -32,7 +32,7 @@ File Description
 	    . "./mk_secure_uimage 0x80800000"
 	        - where 0x80800000 is the DDR location where uImage is
 		  copied to by UBoot, different from platform to platform
-	    . uimage-signed.bin will be generated. It is the uImage
+	    . uimage-signed-pad.bin will be generated. It is the uImage
 	      with signature
 
 Image Layout
@@ -66,5 +66,23 @@ uImage
     |     PADDING       | /
     +-------------------+
 
+DDR Location Parameters
+------------------------------------
+The DDR location parameter provided to "mk_secure_uboot" script is the
+"TEXT_BASE" defined in board/freescale/<plat_name>/config.mk
+
+	MX6Q/DL ARM2 Socket Board		MX6SL ARM2 Socket Board
+	-------------------------		-------------------------
+	0x27800000				0x87800000
+
+The DDR location parameter provided to "mk_secure_uimage" script is the
+"CONFIG_LOADADDR" defined in include/configs/<plat_name>.h (default). And
+can be changed in u-boot command line by "setenv loadaddr=..." with valid
+physical address. The following are the default values
+
+	MX6Q/DL ARM2 Socket Board		MX6SL ARM2 Socket Board
+	-------------------------		-------------------------
+	0x10800000				0x80800000
+
 Please refer to "MX6 Linux High Assurance Boot (HAB) User Guide" for
 detailed test steps.
-- 
1.8.0

